app/test_main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+from app.main import can_access_google_page
```

Comment:
make your imports according to PEP8 convention. Blank lines between group of imports, their order, etc.